pybamm/models/submodels/particle.py

```diff
@@ -42,24 +42,23 @@ def set_differential_system(self, c, j, broadcast=False):
             )
 
         if c.domain[0] == "negative particle":
-            N = -(1 / param.C_n) * pybamm.grad(c)
-            self.rhs = {c: -pybamm.div(N)}
+            N = -pybamm.grad(c)
+            self.rhs = {c: -(1 / param.C_n) * pybamm.div(N)}
             self.algebraic = {}
             self.initial_conditions = {c: param.c_n_init}
+            rbc = -param.C_n * j / param.a_n
```

Comment:
What would we do for a mixed boundary condition of form:
  c + a * dcdx = f
Would we rearrange for c or dcdx?